shh-extras/src/Shh/Prompt.hs

```diff
@@ -9,7 +9,9 @@ import Data.Time
 import Data.List
 
 -- | The type of GHCi prompt functions
-type PromptFn = [String] -> Int -> IO String
+type PromptFn = [String] -- ^ names of the modules currently in scope
```

Comment:
I was just confused about what these arguments mean, so I looked it up in ghc user guide and copied the explanation here.